doc/feature/cppPlugins.md

```diff
@@ -0,0 +1,175 @@
+C++ Plugin API for CPU compressors
+============
+
+This document specifies a modern API for the in-memory C plugin API. It
+will not replace the CPU-GPU transfer plugins.
+
+## Requirements
+
+* Support all features used by current CPU compression plugins
+    * Support 1D and 2D data layouts
+    * Drop alpha during compression of 2D data with alpha
+    * Lossy compression for 2D data
+    * Support for common image pixel layouts
+    * Support for different 1D data (byte, float, double, ...)
+    * Multiple compressors per plugin DSO
+* Information provided for each compressor:
+    * Compression speed (relative to RLE compressor)
+    * Compression ratio (relative to RLE compressor)
```

Comment:
I knew that information is somewhere else. My point is that, despite it may be redundant to repeat it here, nobody would know that by simply reading this file except you and a few others.